managedplugin/logging.go

```diff
@@ -18,6 +18,9 @@ func (c *Client) jsonToLog(l zerolog.Logger, msg map[string]any) {
 		c.metrics.incrementWarnings()
 	case "error":
 		l.Error().Fields(msg).Msg("")
+		if msg["message"].(string) == "table resolver finished with error" && msg["table"].(string) != "" {
```

Comment:
I'm aware that this literal string is a little fragile. It's on a different repo, so I'm open to suggestions 🤷